admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -109,6 +109,112 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
+    display: flex;
+    flex-direction: row;
+    gap: 1.5625rem;
+    padding: 0.625rem;
+    margin: 1rem 0;
+    background: $white;
+    @include border-radius(0.375rem);
+    box-shadow: 0 0 12px rgba(0, 0, 0, 0.15);
+
+    > * {
+      flex-grow: 1;
+    }
+
+    $product-cover-size: 110px;
+
+    .product-header-cover {
+      flex-grow: 0;
+      min-width: $product-cover-size;
+      max-width: $product-cover-size;
```

Comment:
Because the element is in a flex container so its width adapts automatically to the other elements Basically the `width` attribute is not really taken into account So the way I found to limit the size of this element was with min and max

Reply:
Don't lose time with CSS, just make it so it works code wise, we will do the styling and HTML with Valentin if needed. 👍